### ISSUE TITLE
Way to get Portable FF Release to open up in its own window?

### ISSUE THREAD

**Issue by paulri** [darktrojan/openwith#20]
I am using this extension with FF Beta. I have installed the Portable Apps version of the FF Release so that I can quickly look at the current page in the release version. Is there a way to get this portable program/version of FF to open up the page in its own window? When I have FF Beta open, and use this extension, it simply opens up in a new tab, in Beta, which isn't what I want.

**Comment by TekBear** [comment]
I don't think this is possible and I am assuming your using Windows as you did not provide that info. Only one Firefox.exe image can be running on Windows.  When you pass a URL to FFP.EXE (portable Firefox) I think it is going to look for an existing running "Firefox.exe" and pass the URL to that instance. 

In the past when an program enforced 1 instance only the only way to run two instances under the same user you would have to "hack" the binary changing "program.exe" to "progra2.exe" globally. That trick worked for some programs.

**Comment by UksusoFF** [comment]
ORLY? Only one? 
http://portableapps.com/node/3088
 AllowMultipleInstances=true

**Comment by darktrojan** [comment]
Use the `--no-remote` argument if you want two Firefoxen open at once. I don't know if this applies to Portable, but it should.

**Comment by TekBear** [comment]
@UksusoFF:
> ORLY? Only one?
> [http://portableapps.com/node/3088]

That is interesting, the link you mentioned discusses two profiles on the same portable drive. It is not clear they could be run at the same time just reading that post. The changing the "ini" file was suggests single use. It is a hack instead of cloning the entire FirefoxPortable folder.

@darktrojan  has suggested that `--no-remote` here is some further digging that explains this further:
* [http://bit.ly/1KC7FhF], 
* [http://mzl.la/1A79IQE]  

OK after reading that I now I see how you can run two versions of Firefox at same time.  This post  [http://portableapps.com/node/37305] also says it will work but suggests caution with add-ons.  I think if you only install add-ons when one version of Firefox + profile is running you should be fine.  I use `RunLocally=true` setting and that will not work with  `AllowMultipleInstances=true`.

Just to clarify, if I understand this correctly, the solution is start the local firefox version with `--no-remote` and choose any profile except the "default" one (you can always setup a dummy default).  Start the portable version configured with `AllowMultipleInstances=true` which will cause FirefoxPortable.EXE to start Firefox to start with `--no-remote` with the data profile in your ini configuration.  That should solve the issue of two versions of Firefox running one from standard local install and the other from a portable app version. Finally, once you have tested that works using just using shortcuts then you copy your command line and configure "openwith" to launch it the same way.

**Comment by UksusoFF** [comment]
We can't use --no-remote because we need launch FirefoxPortable.exe wrapper. If we run firefox.exe directly it will runned in default mode (not portable).
